roles/confluent.test/molecule/plaintext-rhel/molecule.yml

```diff
@@ -103,6 +103,14 @@ provisioner:
       all:
         scenario_name: plaintext-rhel
 
+        # Testing secrets protection
```

Comment:
probably include this in the plain-rhel scenario, and please add a task to verify.yml that will validate your work